third_party/amd/include/TritonAMDGPUToLLVM/Passes.td

```diff
@@ -32,4 +32,23 @@ def ConvertTritonAMDGPUToLLVM : Pass<"convert-triton-amdgpu-to-llvm", "mlir::Mod
     ];
 }
 
+def ConvertExternToLLVM : Pass<"convert-extern-to-llvm", "mlir::ModuleOp"> {
+    let summary = "Convert Extern Func to LLVM";
+    let description = [{
+
+    }];
+    let constructor = "mlir::triton::createConvertExternToLLVMPass()";
+
+    let dependentDialects = ["mlir::arith::ArithDialect",
+                             "mlir::math::MathDialect",
+                             "mlir::gpu::GPUDialect",
+                             "mlir::scf::SCFDialect",
+                             "mlir::LLVM::LLVMDialect",
+                             "mlir::tensor::TensorDialect",
+                             "mlir::triton::TritonDialect",
+                             "mlir::triton::gpu::TritonGPUDialect",
+                             "mlir::ROCDL::ROCDLDialect"];
```

Comment:
do we really have those dependencies? The pass should be LLVM to LLVM dialect

Reply:
That is true!